pkg/lvm/lvm.go

```diff
@@ -476,7 +481,7 @@ func CreateVG(name string, devicesPattern []string) (string, error) {
 		return name, nil
 	}
 
-	physicalVolumes, err := devices(devicesPattern)
+	physicalVolumes, err := devices(dp)
```

Comment:
i think you should trim leading and trailing whitespaces from every devicePattern

Reply:
https://github.com/metal-stack/csi-driver-lvm/pull/19/files#diff-7c238bacac13d6e6eb36c3fa42a6f44dR447

Reply:
OK, but i meant the resulting parts will also have whitespaces around if for example someone specifies:

```
devicesPattern := " nvme* ,  sda* "
```

Reply:
The code path basically is:

```
	dp := strings.Split(devicesPattern, ",")
	for _, devicePattern := range dp  {
		matches, err := filepath.Glob(strings.TrimSpace(devicePattern))

```
should IMHO be ok so?